dbdimp.c

```diff
@@ -2854,6 +2854,24 @@ SV* dbd_db_FETCH_attrib(SV *dbh, imp_dbh_t *imp_dbh, SV *keysv)
       result= sv_2mortal((newRV_noinc((SV*)hv)));
     }
 
+#if (MYSQL_VERSION_ID >= 50708) && (MYSQL_VERSION_ID <= MARIADB_VERSION_10)
```

Comment:
If this feature is not supported by MariaDB client, then rather check that MARIADB_BASE_VERSION is not defined.

In future Oracle can decide that MySQL version could be increased to 10 or 11 and this code stop working.